lib/core-integration/src/Test/Integration/Scenario/API/Network.hs

```diff
@@ -199,15 +201,20 @@ spec = do
                     HTTP.status404
                     (errMsg404NoEpochNo (T.unpack maxEpochValue))
 
-    it "NETWORK - Can query network clock" $ \ctx ->
-        eventually "ntp status = available" $ do
+    it "NETWORK_CLOCK - Can query network clock" $ \ctx ->
+        eventually "ntp status = (un)available" $ do
             r <- request @ApiNetworkClock ctx
                 Link.getNetworkClock Default Empty
             expectResponseCode @IO HTTP.status200 r
-            verify r
-                [ expectField (#ntpStatus . #status)
-                    (`shouldBe` NtpSyncingStatusAvailable)
-                ]
+            -- network/clock at this point is not supported on Windows
+            if os == "windows"
```

Comment:
What? How? o.O? There's no point having this if it's no supported on Windows.

Reply:
My understanding from discussion with @paweljakubas is that it is currently not supported on Windows in the underneath https://github.com/input-output-hk/ouroboros-network/tree/master/ntp-client, but there are plans to support it in future... So, we can either not have it, or have it and explicitly say that Windows not supported, or use different library (?)

Reply:
Maybe drop the if statement and use `Test.Utils.Windows.pendingOnWindows` here then.

Reply:
done in 0e0f1f3